extensions/answer_summarizers/models.py

```diff
@@ -90,6 +96,52 @@ def _get_top_answers_by_frequency(answers, limit=None):
     ])
 
 
+def _get_top_unresolved_answers_by_frequency(
+        answers_with_classification, limit=None):
+    hashable_answers_with_classification = [{
+        'hashable_answer': _HashableAnswer(ans['answer']),
+        'classification_categorization': ans['classification_categorization']
+    } for ans in answers_with_classification]
+
+    classification_results_dict = {}
+    for ans in hashable_answers_with_classification:
```

Comment:
Seems a bit of a waste to do a for loop in lines 101-104 and then immediately do another in lines 107-116. Might perhaps combine them into one, since the list you're operating with is gonna be quite large.

Reply:
Will do.

Reply:
Done. Merged them into single loop.